editor/src/components/canvas/canvas-strategies/strategies/fragment-like-helpers.ts

```diff
@@ -171,11 +171,20 @@ export const AllFragmentLikeNonDomElementTypes = ['fragment', 'conditional'] as
 export const AllFragmentLikeTypes = [...AllFragmentLikeNonDomElementTypes, 'sizeless-div'] as const
 export type FragmentLikeType = typeof AllFragmentLikeTypes[number] // <- this gives us the union type of the Array's entries
 
+export interface GetElementFragmentLikeTypeOptions {
+  sizelessDivsConsideredFragmentLike: boolean
+}
```

Comment:
Are we likely to add other options here? cc @balazsbajorics

Reply:
we don't have anything planned but this work hasn't started in earnest, so no idea

Reply:
In that case might I suggest we just use a string union param rather than construct a new object every time we call this function. If we do later want to add more options then fair enough, but right now it seems very wrong to introduce an options object for a single option.

Reply:
agreed, fixed in https://github.com/concrete-utopia/utopia/pull/3909/commits/e18e09620f46d7c95e8ff87e45d4596f764596be